curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/create-a-form-element.md

```diff
@@ -37,9 +37,8 @@ assert(
 Your `form` should have an `action` attribute which is set to `https://www.freecatphotoapp.com/submit-cat-photo`
 
 ```js
-assert(
-  $('form').attr('action') === 'https://www.freecatphotoapp.com/submit-cat-photo'
-);
+const action = $('form').attr('action');
+assert(action.match(/https:\/\/(www\.)?freecatphotoapp\.com\/submit-cat-photo/i))
```

Comment:
shouldn't this make more sense using the `test` method? also, the other tests use `test` method

Reply:
and use start and end of string anchors, like the other tests, like this from link-to-external-pages-with-anchor-elements
```
assert(/^https?:\/\/(www\.)?freecatphotoapp\.com\/?$/i.test($('a').attr('href')));
```

```suggestion
assert(/https:\/\/(www\.)?freecatphotoapp\.com\/submit-cat-photo/i.test($('form').attr('action');));
```

Reply:
or just `action` as argument of `test` if you want to keep line 40

Reply:
@nhcarrigan

Reply:
Personally, I'm not too picky about `test` vs `match` - I've been writing a lot of `match` lately so that's what I went with for the suggestion.

Reply:
`match` returns an array, if you are not using that array I think it makes more sense to use `test`
but, whatever works

I would still think the anchors are needed tho
to avoid the tests marking as passing something weird like `action="https://freecatphotoapp.com/submit-cat-photo/sjbkrbgndskjngdksjf"`

Reply:
Yep I do agree the anchors would be a good addition.

Reply:
```suggestion
assert(action.match(/^https:\/\/(www\.)?freecatphotoapp\.com\/submit-cat-photo$/i))
```